packages/common-all/src/util/docutil.ts

```diff
@@ -0,0 +1,7 @@
+import { TextDocument } from "vscode";
```

Comment:
we don't depend on vscode in common-all. we depend on `vscode-url`. see https://github.com/dendronhq/dendron/blob/723d72cf3d1a117faf12296dbe03aed62be8864e/packages/common-all/package.json#L64

also, can we move this into [SchemaUtils](https://github.com/dendronhq/dendron/blob/6e27edf3d7b246e089eb0090222ae3b6a718330e/packages/common-all/src/dnode.ts#L1201) instead?

Reply:
Since this `import { TextDocument } from "vscode";` builds though wouldn't that mean that we have some transitive dependency on vscode? 

Will move to schema utils

Reply:
the reason it works is because we are in a monorepo and node will pick up dependencies by traversing up the file tree looking for `node_modules`. when we try to build this in isolation, `common-all` will not have access to the monorepo `node_modules` and will fail